### PR TITLE
Update Fresh Ingredients feat

### PR DESCRIPTION
**Pull request by elespike**
Fresh Ingredients as printed in PC2: https://2e.aonprd.com/Feats.aspx?ID=6340

### CODE REVIEW

File: packs/feats/fresh-ingredients.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Spend a batch of herbs in wilderness",
+                "label": "Fresh ingredients in the wilderness",
```
Comment (ammalagonc) [inline packs/feats/fresh-ingredients.json:34]: We'll probably want to localize the label.

File: packs/feats/fresh-ingredients.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
             },
             {
                 "key": "FlatModifier",
-                "label": "Spend a batch of herbs",
+                "label": "Fresh ingredients",
```
Comment (ammalagonc) [inline packs/feats/fresh-ingredients.json:47]: Same here

**Review by ammalagonc — CHANGES_REQUESTED** [review]

File: packs/feats/fresh-ingredients.json (hunk 1 above)
Comment (elespike) [reply packs/feats/fresh-ingredients.json:34]: Would you like me to do that within this PR? If so, could you give me an example?

Comment (ammalagonc) [reply packs/feats/fresh-ingredients.json:34]: For a label Syntax option example, check out the Roll Option on Clandestine Cloak (either version).

To add the localization, go to your local repo and find the `re-en.json` file. (Located under pf2e -> static ->lang) and the the entry. Something like this:

```json
"Herbalist" {
    "FreshIngredients": {
         "FlatModifierLabel": "Fresh ingredients"
     },
      "FreshIngredientsWild": {
            "FlatModifierLabel": "Fresh ingredients in the wilderness"
     }
}
```

 Once you add it to the file, just save the file on the branch the changes should populate. On the feat, at the path to the localization (again, refer to Clandestine Cloak's roll option on those are set).

**Review by ammalagonc — APPROVED** [review]
Looks good!

File: packs/feats/fresh-ingredients.json [hunk 3]
```diff
@@ -44,7 +44,7 @@
             },
             {
                 "key": "FlatModifier",
-                "label": "Spend a batch of herbs",
+                "label": "PF2E.SpecificRule.Herbalist.FreshIngredients.FlatModifierLabel",
```
Comment (TikaelSol) [inline packs/feats/fresh-ingredients.json:47]:
```suggestion
```
No need to localize a label that is just the name of the feat.

File: packs/feats/fresh-ingredients.json [hunk 4]
```diff
@@ -31,7 +31,7 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Spend a batch of herbs in wilderness",
+                "label": "PF2E.SpecificRule.Herbalist.FreshIngredientsWilderness.FlatModifierLabel",
```
Comment (TikaelSol) [inline packs/feats/fresh-ingredients.json:34]:
```suggestion
                "label": "PF2E.SkillVariant.Wilderness",
```
I'll add this localization in another update

File: static/lang/re-en.json [hunk 5]
```diff
@@ -3942,6 +3942,14 @@
...
             },
+            "Herbalist": {
+                "FreshIngredients": {
+                    "FlatModifierLabel": "Fresh Ingredients"
+                },
+                "FreshIngredientsWilderness": {
+                    "FlatModifierLabel": "Fresh Ingredients in the wilderness"
+                }
+            },
             "HeroPointDeck": {
```
Comment (TikaelSol) [inline static/lang/re-en.json:3953]:
```suggestion
            },
            "HeroPointDeck": {
```

**Pull request merged**